mitreattack/attackToExcel/stixToDf.py

```diff
@@ -479,31 +484,29 @@ def matricesToDf(src, domain):
         merge is a list of CellRange objects that need to be merged for formatting of the sub-techniques in the matrix
         columns is the number of columns in the data
     """
-
-    print("building matrices... ", end="", flush=True)
     matrices = src.query([Filter("type", "=", "x-mitre-matrix")])
     matrices = remove_revoked_deprecated(matrices)
     matrices_parsed = []
 
-    for matrix in matrices:
+    for matrix in tqdm(matrices, desc="parsing matrices"):
```

Comment:
I'll keep that in mind! At the moment, I feel like I'd have to reorganize a bit of code to make it work in a logical way with a manual update, so I'm going to leave it as is for the moment, since you said you were okay with that.